src/components/content/content.pw.tsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import ContentComponent from "./components.test-pw";
+import { CHARACTERS } from "../../../cypress/support/component-helper/constants";
+import {
+  contentBody,
+  contentTitle,
+} from "../../../playwright/components/content";
+import {
+  checkAccessibility,
+  getStyle,
+} from "../../../playwright/support/helper";
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+const totalWidth = 1366;
+const alignmentData = ["left", "center", "right"];
+
+test.describe("check Content component properties", () => {
+  [
+    ["primary", "rgba(0, 0, 0, 0.9)"],
+    ["secondary", "rgba(0, 0, 0, 0.55)"],
+  ].forEach(([variant, titleColor]) => {
+    test(`should check ${variant} as variant for Content components`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ContentComponent variant={variant} />);
+      const contentElement = contentTitle(page);
+      await expect(contentElement).toHaveCSS("color", `${titleColor}`);
+    });
+  });
+
+  testData.forEach((children) => {
+    test(`should check ${children} as children for Content component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <ContentComponent title="Title">{children}</ContentComponent>
+      );
+
+      await expect(await contentBody(page)).toHaveText(children);
+    });
+  });
+
+  testData.forEach((title) => {
+    test(`should check ${title} as title for Content component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ContentComponent title={title} />);
+
+      await expect(await contentTitle(page)).toHaveText(title);
+    });
+  });
+
+  test(`should check inline parameter is enabled for Content component`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<ContentComponent inline />);
+
+    const contentElement = contentTitle(page);
+    await expect(contentElement).toHaveCSS("display", "inline-block");
+  });
+
+  alignmentData.forEach((alignment) => {
+    test(`should check ${alignment} as alignment for Content component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ContentComponent align={alignment} />);
+
+      const contentElement = contentTitle(page);
+      await expect(contentElement).toHaveCSS("text-align", alignment);
+    });
+  });
+
+  ([
+    [70, (totalWidth * 70) / 100],
+    [50, (totalWidth * 50) / 100],
+  ] as [number, number][]).forEach(([titleWidth, computedWidth]) => {
+    test(`should check titleWidth as ${titleWidth} for Content component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<ContentComponent titleWidth={titleWidth} />);
+      const cssWidth = await getStyle(contentTitle(page), "width");
+      await expect(parseInt(cssWidth)).toBeLessThanOrEqual(computedWidth - 30);
```

Comment:
**issue**: unnecessary `await` x2

Reply:
**suggestion**: remove the unnecessary awaits, like so `expect(parseInt(cssWidth)).toBeLessThanOrEqual(computedWidth - 30) expect(parseInt(cssWidth)).toBeGreaterThanOrEqual(computedWidth - 31 );`